Formula/anycable-go.rb

```diff
@@ -15,8 +15,16 @@ class AnycableGo < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "build", "-mod=vendor", "-ldflags", "-s -w -X main.version=#{version}",
-                          "-trimpath", "-o", "#{bin}/anycable-go",
+    build_vars =
+      if build.head?
+        "github.com/anycable/anycable-go/utils.sha=#{version.commit}"
+      else
+        "main.version=#{version}"
+      end
+
+    system "go", "build", "-mod=vendor", "-ldflags",
+                          "-s -w -X #{build_vars}",
+                          *std_go_args,
```

Comment:
I would tune it a bit, just to keep `-X` closer with `build_vars`, something like this:

```suggestion
    ldflags = %w[
      -s -w
    ]
    ldflags << if build.head?
      "-X github.com/anycable/anycable-go/utils.sha=#{version.commit}"
    else
      "-X main.version=#{version}"
    end

    system "go", "build", "-mod=vendor", "-ldflags", ldflags.join(" "), *std_go_args,
```